adminlogin.aspx.cs

```diff
@@ -9,9 +11,45 @@ namespace LogBook
 {
     public partial class adminlogin : System.Web.UI.Page
     {
+        string strcon = ConfigurationManager.ConnectionStrings["con"].ConnectionString;
+
         protected void Page_Load(object sender, EventArgs e)
         {
 
         }
+
+        protected void Button1_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                SqlConnection con = new SqlConnection(strcon);
+                if (con.State == System.Data.ConnectionState.Closed)
+                {
+                    con.Open();
+                }
+                SqlCommand cmd = new SqlCommand("select * from admin_login_tbl where username='" + TextBox1.Text.Trim() + "' AND password='" + TextBox2.Text.Trim() + "'", con);
+                SqlDataReader dr = cmd.ExecuteReader();
+                if (dr.HasRows)
+                {
+                    while (dr.Read())
+                    {
+                        //Response.Write("<script>alert('" + dr.GetValue(0).ToString() + "');</script>");
+                        Session["username"] = dr.GetValue(0).ToString();
```

Comment:
### You can set the value to be the same column name for your DataTable

You can explicitly define the username as dr["your_table_column_name"].ToString() instead of using GetValue(x);
Nobody will know what GetValue() is unless they know which columns are represented in each index.

```suggestion
                        Session["username"] = dr["username"].ToString();
```